### PR TITLE
Added clip creator to the Twitch clip response

### PR DESCRIPTION
**Pull request by zneix**
Looks like this:
TRIHARD 7

### CODE REVIEW

**Comment by alazymeme** [comment]
4House

**Comment by Mm2PL** [comment]
Thank you for your valuable comment on this pull request, AlazyMeme.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Unreleased
 
-- Add support for FrankerFaceZ emote links
+- Added support for FrankerFaceZ emote links. (#57)
+- Added author name to TWitch clips response. (#76)
```
Comment (leon-richardt) [inline CHANGELOG.md:4]:
```suggestion
- Added author name to Twitch clips response. (#76)
```

**Review by leon-richardt — CHANGES_REQUESTED** [review]

File: internal/resolvers/twitch/resolver.go [hunk 2]
```diff
@@ -17,10 +17,11 @@ import (
 const (
 	twitchClipsTooltipString = `<div style="text-align: left;">
 <b>{{.Title}}</b><hr>
+<b>Author:</b> {{.AuthorName}}<br>
```
Comment (leon-richardt) [inline internal/resolvers/twitch/resolver.go:20]: Personally, I think something like "Clipped by:" would be nicer here. But maybe that's up to personal taste

Comment (zneix) [reply internal/resolvers/twitch/resolver.go:20]: I thought following already common one-word format would be nice, but I guess your suggestion makes more sense, Though I guess I'd leave it up to pajlada to decide which one fits better.

Comment (pajlada) [reply internal/resolvers/twitch/resolver.go]: Went with "Clipped by" :+1:

**Review by pajlada — APPROVED** [review]

**Pull request merged**